proto/api/v1/policy_service.proto

```diff
@@ -126,6 +126,14 @@ message GetPolicyMitreVectorsResponse {
   repeated storage.MitreAttackVector vectors = 2;
 }
 
+message ExportAsCustomResourcesRequest {
```

Comment:
I will avoid the word Export. Rcommend using "SaveAs"

SaveAsCustomResourceRequest (resource: singular)